druid/src/tests/mod.rs

```diff
@@ -14,7 +14,7 @@
 
 //! Additional unit tests that cross file or module boundaries.
 
-mod harness;
+pub(crate) mod harness;
```

Comment:
If this was `pub` then would other crates be able to use this of tests?
I would really like to be able to use the harness in my own apps that use druid.

Reply:
That makes sense, Thanks.